### PR TITLE
Add support for Sidekiq 7

### PR DESCRIPTION
**Pull request by lddelchev**

### CODE REVIEW

**Review by katya-spasova — APPROVED** [review]

File: lib/instana/util.rb [hunk 1]
```diff
@@ -161,6 +161,24 @@ def header_to_id(given)
...
       end
+
+      # Get the redis url from redis client
+      #
+      # @param client [String] the Redis client
+      #
+      # @return [String, nil]
+      #
+      def get_redis_url(client)
+        if client.respond_to?(:config) # redis-client
```
Comment (vdimchev) [inline lib/instana/util.rb:172]: Are the things in the comment different redis libraribes?

I would probably also add which check to which redis version is related.

**Review by vdimchev — APPROVED** [review]
Looks good!

File: lib/instana/util.rb (hunk 1 above)
Comment (lddelchev) [reply lib/instana/util.rb:172]: These are different redis client libraries. Until 2017 Sidekiq used `hi-redis`, after that they switched to `redis-rb` and recently in version 7 they started using `redis-client`

File: lib/instana/util.rb [hunk 2]
```diff
@@ -161,6 +161,30 @@ def header_to_id(given)
         return '' unless given.match(/\A[a-z\d]{16,32}\z/i)
         given
       end
+
+      # Get the redis url from redis client.
+      # In different versions of Sidekiq different redis clients are use, so getting the redis url should be
```
Comment (vdimchev) [inline lib/instana/util.rb:166]: `are use` -> `are used`

**Review by vdimchev — APPROVED** [review]

**Pull request merged**